### ISSUE TITLE
Form "notitle": true should not remove <label>

### ISSUE THREAD

**Issue by madhaze** [jsonform/jsonform#321]
Removing `<label>` causes accessibility scans to fail. 
For accessibility `<input>`s need to have `<label>` or can use aria-label.  https://www.w3.org/WAI/tutorials/forms/labels/

right now if `"notitle": true` is set the <label> element does not render.
```  
"form": [
    {
      "key": "field",
      "notitle": true
    },
    {
      "type": "submit",
      "title": "Submit"
    }
  ]
```
```
<div class="controls">
  <input type="text" class="form-control" name="field" value="" id="jsonform-19-elt-field">
  <span class="help-block jsonform-errortext" style="display:none;"></span>
</div>
```

It adding a class like bootstraps `sr-only` to the `<label>` or adding `aria-label` attribute to the `<input>` would help make the form more accessible.

Below is the return from doing a test on https://jsonform.github.io/jsonform/playground/index.html?example=gettingstarted with the "notitle": true set.

>Title: WCAG 1.3.1,WCAG 4.1.2: Ensures every form element has a label (#jsonform-20-elt-field)
Tags: Accessibility, WCAG 1.3.1, WCAG 4.1.2, label
>
>Issue: Ensures every form element has a label (label - https://accessibilityinsights.io/info-examples/web/label)
>
>Target application: JSON Form Playground - https://jsonform.github.io/jsonform/playground/index.html?example=gettingstarted
>
>Element path: #jsonform-20-elt-field
>
>Snippet: <input type="text" class="form-control" name="field" value="" id="jsonform-20-elt-field">
>
>How to fix: 
>Fix any of the following:
>  aria-label attribute does not exist or is empty
>  aria-labelledby attribute does not exist, references elements that do not exist or references >elements that are empty
>  Form element does not have an implicit (wrapped) <label>
>  Form element does not have an explicit <label>
>  Element has no title attribute
>  Element has no placeholder attribute
>  Element's default semantics were not overridden with role="none" or role="presentation"
>
>Environment: Chrome version 88.0.4324.146
>
>====
>
>This accessibility issue was found using Accessibility Insights for Web 2.23.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at >http://aka.ms/AccessibilityInsights.

**Comment by tchapi** [comment]
Hi @madhaze and thanks for the report. I think the `aria-label` route seems the most logical and compatible. Would you be able to make a PR for this change ?
